lib/middleware/common.js

```diff
@@ -117,7 +120,7 @@ var createServeFile = function (fs, directory, config) {
       responseData = (transform && transform(data.toString())) || data
 
       if (rangeHeader) {
-        var code = convertForRangeRequest()
+        const code = convertForRangeRequest()
```

Comment:
You can update L122-L127 into `response.writeHead(rangeHeader ? convertForRangeRequest() : 200)`

Reply:
done